Reloader/Xamarin.Forms.HotReload.Reloader/Xamarin.Forms.HotReload.Reloader.csproj

```diff
@@ -2,10 +2,22 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <!-- Optional: Publish the repository URL in the built .nupkg (in the NuSpec <Repository> element) -->
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+     
+    <!-- Optional: Embed source files that are not tracked by the source control manager in the PDB -->
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+      
+    <!-- Optional: Build symbol package (.snupkg) to distribute the PDB containing Source Link -->
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <!-- Optional: Include the PDB in the built .nupkg -->
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Xamarin.Forms" Version="3.6.0.220655" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-*" PrivateAssets="All"/>    
```

Comment:
Won't it appear in Release builds ?

Reply:
no, as I quoted `PrivateAssets="All"` marks it as development dependency

Reply:
this package creates the JSON file with github urls and adds those json to the pdb. During debug, VS2017 detects the JSON and fetches the cs files from github to step into your lib.

Reply:
I don't know if VS for MacOS supports it, but VS2017 does support it since 15.5

Reply:
this is added to the PDB:

```
<SourceLink>
    <Conversion FilePath="D:\HotReload_MA1981\*" Uri="https://raw.githubusercontent.com/MagicAndre1981/HotReload/38584b9e19bc02026e68ca68eb8ac3bf2aa0bc62/*" />
  </SourceLink>
```

so that debugger finds the source